exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -109,12 +111,11 @@
    * in reverse order, starting from the {@code heightMax}.
    * @param count Number of blocks to return.
    *        It should be in range [1, {@linkplain ExonumApi#MAX_BLOCKS_PER_REQUEST}]
-   * @param skipEmpty if {@code true}, then only non-empty blocks will be returned
+   * @param blockFilter blocks filtering option identifies to skip or include empty blocks
    * @param heightMax maximum height of the returned blocks.
    *        If the {@code heightMax} is greater than actual blockchain height then
    *        the actual height will be used
-   * @param withTime if {@code true}, then includes block commit times in the response.
-   *        See {@linkplain Block#getCommitTime()}.
+   * @param timeOption block commit time option identifies to include or not block commit time.
```

Comment:
@param timeOption controls whether to include the block commit time (no dot)?

Reply:
dot is necessary because it has a few sentences below